generators/chipyard/src/main/scala/ConfigFragments.scala

```diff
@@ -60,6 +67,10 @@ class WithNoUART extends Config((site, here, up) => {
   case PeripheryUARTKey => Nil
 })
 
+class WithNoSPIFlash extends Config((site, here, up) => {
```

Comment:
Is this going to be rebased on #544 so that you can remove this? Or is the plan to merge this in before?

Reply:
Depends on which goes in first. If #544 goes in first, yea, I'll remove this. If this goes in first, I'll make the change and commit it to the #544 branch. I imagine the former will happen.